nodeup/pkg/model/containerd.go

```diff
@@ -461,6 +461,13 @@ func (b *ContainerdBuilder) buildCNIConfigTemplateFile(c *fi.NodeupModelBuilderC
     ]
 }
 `
+
+	routes := `[{ "dst": "0.0.0.0/0" }]"`
```

Comment:
Not blocking, but I dislike putting JSON strings in code. It's more maintainable if you call `json.Marshal()`.

Reply:
Yes, I agree and I did debate structifying this, but I don't think it has a well-defined schema, much less a go type we can import - so it would be a big change.   Or I could start with routes and we can gradually expand, but then it's weird that we are mixing approaches here.

Reply:
You can use ad-hoc types like `[]map[string]string{{"dst":"0.0.0.0/0"}}` or something like that.

Reply:
Done